bridge/main_darwin.go

```diff
@@ -1,9 +1,24 @@
 package bridge
 
-import "github.com/progrium/macdriver/cocoa"
+import (
+	"github.com/progrium/macdriver/cocoa"
+	"github.com/progrium/macdriver/core"
+	"github.com/progrium/macdriver/objc"
+)
 
 func Main() {
 	app := cocoa.NSApp()
+	nsbundleMain := cocoa.NSBundle_Main()
+	nsbundle := nsbundleMain.Class()
+	nsbundle.AddMethod("__bundleIdentifier", func(self objc.Object) objc.Object {
+		if self.Pointer() == nsbundleMain.Pointer() {
+			return core.String("com.progrium.shellbridge")
```

Comment:
@progrium do you have another preference on the identifier here?

Reply:
good enough